rs/state_manager/src/lib.rs

```diff
@@ -2970,10 +3003,10 @@ impl StateManager for StateManagerImpl {
     ///    the latest checkpoint ≤ H passed to `remove_states_below`.
     ///  * *LSH* stands for "Latest State Height". This is the latest state that
     ///    the state manager has.
-    ///  * *LCH* stands for "Latest Checkpoint Height*. This is the height of
+    ///  * *LCH* stands for "Latest Checkpoint Height". This is the height of
```

Comment:
Should this be the latest verified checkpoint height?

Reply:
Good point. This also applies to `*OCK* standing for "Oldest verified Checkpoint to Keep"`